pkg/scheduler/preemption/preemption.go

```diff
@@ -94,8 +94,8 @@ func (p *Preemptor) GetTargets(wl workload.Info, assignment flavorassigner.Assig
 	// trying to preempt more own workloads and borrow at the same time.
 
 	if len(sameQueueCandidates) == len(candidates) {
-		// There is no risk of preemption of workloads from the other queue,
-		// so we can try borrowing.
+		// There is no need of preemption of workloads from other queues,
```

Comment:
nit: I would suggest to revert unrelated changes

Reply:
```suggestion
		// There is no possible preemption of workloads from other queues,
```

Reply:
Done.